demo/src/app/components/+timepicker/timepicker-section.list.ts

```diff
@@ -82,6 +83,16 @@ export const demoComponentContent: ContentSection[] = [
         html: require('!!raw-loader?lang=markup!./demos/custom-validation/custom-validation.html'),
         outlet: DemoTimepickerCustomValidationComponent
       },
+      {
+        title: 'IsValid',
+        anchor: 'isvalid',
+        component: require('!!raw-loader?lang=typescript!./demos/isvalid/isvalid'),
+        html: require('!!raw-loader?lang=markup!./demos/isvalid/isvalid.html'),
+        description: `
+        <p>Illustrates emitting true if time value is a valid. You have to enter invalid time to see error.</p>
```

Comment:
`isValid` event emits true if a value is a valid date. Enter an invalid date to see the error.